base/applications/fontview/lang/de-DE.rc

```diff
@@ -10,19 +10,19 @@ BEGIN
     IDS_ERROR_NOMEM "Es steht nicht genügend Speicher zur Verfügung."
     IDS_ERROR_NOFONT "Die angegebene Datei %1 ist keine gültige Schriftartendatei."
     IDS_ERROR_NOCLASS "Fehler beim Initialisieren der Fensterklasse."
-    IDS_ERROR_ISINSTALLED "This font is already installed!" 
-    IDS_ERROR_FONTCPY "Failed to copy the font file!"
-    IDS_ERROR_OPENKEY "Failed to open the fonts key!"
-    IDS_ERROR_REGISTER "Failed to register the new font!"
-    IDS_SUCCESS "Success"
-    IDS_COMPLETED "Font installation completed."
-    IDS_FILTER_LIST "All Supported Fonts (*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc)\0*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc\0\
+    IDS_ERROR_ISINSTALLED "Die Schriftart ist schon installiert!" 
+    IDS_ERROR_FONTCPY "Kopieren der Schriftartdatei fehl geschlagen!"
+    IDS_ERROR_OPENKEY "Öffnen des Schriftart Keys fehl geschlagen!"
+    IDS_ERROR_REGISTER "Registrierung der neuen Schriftart fehl geschlagen!"
+    IDS_SUCCESS "Erfolg"
+    IDS_COMPLETED "Installation der Schriftarten vollständig."
+    IDS_FILTER_LIST "Alle unterstützten Schriftarten (*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc)\0*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc\0\
 Font File (*.fon;*.fnt)\0*.fon;*.fnt\0\
 TrueType Font (*.ttf)\0*.ttf\0\
 TrueType Font Collection (*.ttc)\0*.ttc\0\
 OpenType Font (*.otf)\0*.otf\0\
 OpenType Font Collection (*.otc)\0*.otc\0\
 All Files (*.*)\0*.*\0"
-    IDS_PREVIOUS "< P&revious"
-    IDS_NEXT "&Next >"
+    IDS_PREVIOUS "< V&orheriges"
```

Comment:
Can you tell me how to reach that button in Windows fontview? I do want to quickly double-check whether &o is the correct accelerator here.